woocommerce/woocommerce-android
# Add A/B experiment to test new Jetpack tunnel timeout/retry policy

A/B test for: #7254 


This PR requires[ this FluxC changes](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2520) before merged

### Description
Adding a new variant to test success ratio for site verification in `SitePicker` screen. The 2 options are: 
- `CONTROL` -> Keep using the Jetpack tunnel timeout of 30s 
- `NEW_TIMEOUT_POLICY` -> Reduce the timeout to 15s for site verification request. 

I created an [new experiment in Firebase](https://console.firebase.google.com/u/0/project/api-project-108380595987/config/experiment/results/8) for this. Right now, is enabled for 100% in debug mode. Once code is merge I'll edit it to enable the experiment to `release` builds.

### Testing instructions

Since this A/B test has no visible impact in the UX the most straightforward way to test this is just by checking that the returned value from firebase remote config changes between different app installations. For that follow this steps: 
- Install Woo from scratch and log in
- In site picker screen before clicking on continue place a breakpoint in `SitePickerRepository.verifySiteWooAPIVersion` function (line 65). 
- Set the app to Debug and click "Continue" button from SitePicker. 
- Verify the value for `overrideRetryPolicy` param. 
- Check this value changes for different installations.
